graf3d/x3d/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ set(libname X3d)
 
 include_directories(${X11_INCLUDE_DIR})
 
-ROOT_GENERATE_DICTIONARY(G__X3D TViewerX3D.h TX3DFrame.h MODULE ${libname} LINKDEF LinkDef.h DEPENDENCIES Graf3d Gui)
```

Comment:
@amadio, I feel fairly strongly about this. If we aim to improve something we have to go the last mile.

@Teemperor, then let's replace this with `ROOT_STANDARD_LIBRARY_PACKAGE`. I think we can afford to wait for a few days and fix this once and for all.